Control coverage/Feature/APIManagement.md

```diff
@@ -11,7 +11,7 @@
 - [Azure_APIManagement_AuthZ_Validate_JWT](#azure_apimanagement_authz_validate_jwt)
 - [Azure_APIManagement_AuthN_Disable_Management_API](#azure_apimanagement_authn_disable_management_api)
 - [Azure_APIManagement_AuthZ_Enable_User_Authorization_For_API](#azure_apimanagement_authz_enable_user_authorization_for_api)
-- [Azure_APIManagement_AuthN_Use_AAD_for_Client_AuthN](#azure_apimanagement_authn_use_aad_for_client_authn)
+- [Azure_APIManagement_AuthN_Use_Microsoft_Entra_ID_for_Client_AuthN](#azure_apimanagement_authn_use_microsoft_entra_id_for_client_authn)
```

Comment:
update later reference for this control in same ffile

Reply:
done.